modules/Components/src/client/ACASFormFields.coffee

```diff
@@ -324,14 +326,14 @@ class window.ACASFormLSNumericValueFieldController extends ACASFormAbstractField
 			@$('.bv_units').html @getModel().get('unitKind')
 		super()
 
-class window.ACASFormLSCodeValueFieldController extends ACASFormAbstractFieldController
+class ACASFormLSCodeValueFieldController extends ACASFormAbstractFieldController
 	###
 		Launching controller must:
 		- Initialize the model with an LSValue
     Do whatever else is required or optional in ACASFormAbstractFieldController
 	###
 	events: ->
-		_.extend {}, super,
+		_.extend {}, super(),
```

Comment:
This wasn't working. I'm not sure why but adding `super()` instead of just `super` fixed it. I think this might be a coffeescript2 compiler change